lib/recognize-stream.ts

```diff
@@ -14,62 +14,21 @@
  * limitations under the License
  */
 
-import { OutgoingHttpHeaders } from 'http';
 import { Authenticator, contentType, qs } from 'ibm-cloud-sdk-core';
 import omit = require('object.omit');
 import pick = require('object.pick');
 import { Duplex, DuplexOptions } from 'stream';
 import { w3cwebsocket as w3cWebSocket } from 'websocket';
+import SpeechToTextV1 = require('../speech-to-text/v1-generated');
```

Comment:
Is there a reason you're pulling this import from the generated module and not the adapter module?

Reply:
The autocomplete giveth, the autocomplete taketh away.

Fixed, and used the proper class for the inheritance on `RecognizeStream.Options`.